serve/filters/manager.go

```diff
@@ -62,6 +62,13 @@ func (f *Manager) NewBlockFilter() string {
 	return f.filters.newFilter(blockFilter)
 }
 
+// NewTransactionFilter creates a new Transaction filter, and returns the corresponding ID
+func (f *Manager) NewTxFilter() string {
```

Comment:
Why not add support for tx filter options?

Reply:
Thanks, I missed this one.
Added filter options to apply to this commit. [0a00823](https://github.com/gnolang/tx-indexer/pull/36/commits/0a00823a099763b4592a0ff5a3cbc4007320d195) 

It would be nice if the options in the filter worked similarly to the options in GraphQL.
I'll contribute to add documentation and missing options.